modules/containers-signature-verify-skopeo.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-container-signature.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="containers-signature-verify-skopeo_{context}"]
+= Using skopeo to verify signatures of Red Hat container images
+
+You can verify the signatures for container images included in an OCP release image by pulling those signatures from link:https://mirror.openshift.com/pub/openshift-v4/signatures/openshift-release-dev/ocp-release/[ocp-release]. Because the signatures on the mirror site are not in a format readily understood by podman or CRI-O, you can use the `skopeo standalone-verify` command to verify that the your release images are signed by Red Hat.
+
+.Prerequisite
+* You have installed the `skopeo` command-line utility.
+
+.Procedure
+
+. Get the full SHA for your release by running the following command:
++
+[source,terminal]
+----
+$ oc admin release info <release_version>
+----
+Substitute <release_version> with your release number, for example, `4.12.23`.
++
+.Example output
++
+[source,terminal]
+----
+Pull From: quay.io/openshift-release-dev/ocp-release@sha256:2309578b68c5666dad62aed696f1f9d778ae1a089ee461060ba7b9514b7ca417 -
+----
+//Q: is this what we should see if we run the oc admin release info command?
+
+. Pull down the Red Hat release key by running the following command:
++
+[source,terminal]
+----
+$ curl -o pub.key https://access.redhat.com/security/data/fd431d51.txt
+----
+//Q: is there a user value to enter here that our admin needs to get from somewhere?
+
+. Get the signature file for the specific release that you want to verify by running the following command:
++
+[source,terminal]
+----
+$ curl -o signature-1 https://mirror.openshift.com/pub/openshift-v4/signatures/openshift-release-dev/ocp-release/sha256%3D<sha_from_version>/signature-1
```

Comment:
I'm going to suggest using attributes here because of the following command. You can search for "\ <1>" in our docs to see examples

```suggestion
$ curl -o signature-1 https://mirror.openshift.com/pub/openshift-v4/signatures/openshift-release-dev/ocp-release/sha256%3D<sha_from_version>/signature-1<1>
```